server/clearbit.js

```diff
@@ -154,32 +164,32 @@ export default class Clearbit {
     const domain = getDomain(user);
 
     if (!this.client || !discover_enabled || _.isEmpty(discover_segments)) {
-      this.debug("Discover not enabled", { discover_segments });
+      this.logSkip(user, "discover", "Discover not enabled", { discover_segments });
```

Comment:
@mkawalek - let's do `const asUser = hull.asUser()` at the top of that function and then call the logger directly here in each of those places please.